docs/sdk-and-tools/indices/accounts.md

```diff
@@ -0,0 +1,58 @@
+---
+id: es-index-accounts
+title: Elasticsearch index: accounts
+---
+
+The `_id` field for this index is composed of bech32 encoded address.
+
+| Field      | Description                                                                                                |
+|------------|------------------------------------------------------------------------------------------------------------|
+| address    | The address field holds the address bytes in a bech32 encoding.                                            |
+| balance    | The balance field holds the amount of EGLD the address has.                                                |
+| balanceNum | The balanceNum field holds the amount of EGLD the address has in a numeric format(denominated with 10^18). |
+| nonce      | The nonce field represents the sequence number of the address.                                             |
+| shardID    | The shardId field represents in which shard the address belongs.                                           |
```

Comment:
The shardID field represents the shard the address belongs to, based on its bytes.

Reply:
done